src/concrete/ml/sklearn/base.py

```diff
@@ -1280,8 +1280,41 @@ def __init__(self, n_bits: int):
         #: The model's inference function. Is None if the model is not fitted.
         self._tree_inference: Optional[Callable] = None
 
+        #: Set to `True` to enable the rounding feature, or `False` to disable it.
```

Comment:
small detail but this is not something we "set" here, I would rephrase this (like `Boolean that indicates if the rounding feature should be used during fitting` or similar)